.gitignore

```diff
@@ -103,6 +103,19 @@ venv.bak/
 .spyderproject
 .spyproject
 
+# VScode 
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+!.vscode/*.code-snippets
+
+# Local History for Visual Studio Code
+.history/
+
+# Built Visual Studio Code Extensions
+*.vsix
```

Comment:
Please add these to the global .gitignore file: [catalystneuro/.github/.gitignore](https://github.com/catalystneuro/.github)